apps/sms/services/test/unit/moz_mobile_message/messaging_service_test.js

```diff
@@ -0,0 +1,670 @@
+/*global bridge,
+         MocksHelper
+*/
+
+'use strict';
+
+require('/services/js/bridge_service_mixin.js');
+require('/services/test/unit/mock_bridge.js');
+require('/views/shared/test/unit/mock_messages.js');
+require('/views/shared/test/unit/mock_settings.js');
+require('/views/shared/test/unit/mock_smil.js');
+
+var MocksHelperForAttachment = new MocksHelper([
+  'bridge',
+  'Settings',
+  'SMIL'
+]).init();
+
+var MessagingService, successEvt, errorEvt;
+
+suite('Messaging service >', function() {
+
+  MocksHelperForAttachment.attachTestHelpers();
+
+  suiteSetup(function(done) {
+    window.self = {};
+    window.importScripts = sinon.stub();
+
+    require('/services/js/moz_mobile_message/messaging_service.js', done);
```

Comment:
I think you can require the js at the top ? and I'm not sure we need to mock importScripts... or maybe you can mock it and make it throw because it should not be called :)